assets/pages/contributions-thankyou/monthlyContributionsPending.jsx

```diff
@@ -0,0 +1,68 @@
+// @flow
```

Comment:
If that is the case, is is possible to wrap everything in a component and pass the copy as an argument? In that way, we will be avoiding of repeating ourselves.